### PR TITLE
Fix ConcurrentModificationException

### PR DESCRIPTION
**Pull request by Messna**
Fix ConcurrentModificationException when the Pokemon-List is updated while being iterated through.

### CODE REVIEW

File: src/main/java/dekk/pw/pokemate/tasks/EvolvePokemon.java [hunk 1]
```diff
@@ -33,7 +34,8 @@
     @Override
     public void run(Context context) {
         try {
-            for (Pokemon pokemon : context.getApi().getInventories().getPokebank().getPokemons()) {
+            List<Pokemon> pokemonList = context.getApi().getInventories().getPokebank().getPokemons();
```
Comment (null-dev) [inline src/main/java/dekk/pw/pokemate/tasks/EvolvePokemon.java:37]: This literally change nothing, you are using the **same** instance of the list. The problem is **inside the list**, not with the reference to the list.

Replace this line with:
`List<Pokemon> pokemonList = new ArrayList<>(context.getApi().getInventories().getPokebank().getPokemons());`.

That would actually fix the issue as it clones the list and creates a **new** instance of the list.

_Remember to import ArrayList!_

**Comment by Messna** [comment]
Oops sorry you are right, crashed 3 times before the change and then had it running for an hour without an issue, stupid luck based bugs that force you to know what you are doing... ;) 
I will send an update tomorrow.

File: src/main/java/dekk/pw/pokemate/tasks/EvolvePokemon.java (hunk 1 above)
Comment (SwipeX) [inline src/main/java/dekk/pw/pokemate/tasks/EvolvePokemon.java:37]: @null-dev very true. I just need to have everything to be synced.

Comment (SwipeX) [inline src/main/java/dekk/pw/pokemate/tasks/EvolvePokemon.java:37]: The array list is not preferrable due to memory.

**Comment by SwipeX** [comment]
Ok, just rebase then, we will scrap this and wait for your next commit.

**Pull request closed without merging**